findContact/FindContactModal.js

```diff
@@ -208,12 +289,12 @@ class FindContactModal extends React.Component {
 }
 
 FindContactModal.propTypes = {
-  packageInfo: PropTypes.object,
-  stripes: PropTypes.object.isRequired,
+  closeCB: PropTypes.func.isRequired,
   mutator: PropTypes.object.isRequired,
   resources: PropTypes.object.isRequired,
-  closeCB: PropTypes.func.isRequired,
-  addContacts: PropTypes.func.isRequired,
```

Comment:
why don't you just use this function? It adds contacts as expected

Reply:
Updated